src/common/imagpng.cpp

```diff
@@ -342,6 +343,35 @@ wxPNGImageData::DoLoadPNGFile(wxImage* image, wxPNGInfoStruct& wxinfo)
         return;
 
     png_read_image( png_ptr, lines );
+
+    // load "Description" text chunk
+    png_textp text_ptr;
+    int num_comments = png_get_text( png_ptr, info_ptr, &text_ptr, nullptr );
```

Comment:
Extremely minor, but it can be nice to show that this doesn't change

```suggestion
    const int num_comments = png_get_text( png_ptr, info_ptr, &text_ptr, nullptr );
```